### DISCUSSION TITLE
Upgrade/Update path for private solution 3.2 -> 3.3

### DISCUSSION THREAD

**Discussion by omerzubair** [Azure/ipam#304] [Show and tell]
This is simply a confirmation on how buttery smooth process was update.ps1 to go from v3.2.0 -> v3.3.0.
Following are the steps:

1. I simply made ACR to enable from "all networks"
2. then ran following command.

`- update.ps1 -AppName <name of IPAM app service> -ResourceGroupName <name of IPAM Resource Group>”`

3. After upgrade you can check the version upgrade by navigating to your website 

`- "https://<app service fqdn here>/api/status"`

4. Closed the ACR to private endpoint only. 

Thanks to @DCMattyG

**Comment by DCMattyG** [comment]
Thank you so much for the wonderful feedback @omerzubair! Always happy to help make your life easier however I can 😉
